### ISSUE TITLE
Crash on users with no password

### ISSUE THREAD

**Issue by roland-github** [cagdasbas/ipa_notify#6]
Having a ipa group which contains user with only key based authentication will fail as they do not have password and password expiry.
A try would be nice around this issue:

```
                        try:
                                email = user['mail'][0]
...
...
                                                except ValueError:
                                                        logging.error("email send error, aborting...")
                                                        break
                        except KeyError as err:
                                logging.error("no expiration or mail for %s", user['uid'][0])
                                continue
```

**Comment by cagdasbas** [comment]
Hi roland!
It was a busy week and I'll take look into as soon as I can.

**Comment by cagdasbas** [comment]
Sorry for the delay. Fixed in 0.3.12. Please feel free to open open it again.
